bin/container.sh

```diff
@@ -129,14 +125,13 @@ stop_container() {
   [ "${cid}" != "" ] && docker stop ${cid}
 }
 
-
 remove_container() {
   [ ${1} ] && container=${1}
 
   if [ ${container} ]; then
     stop_container ${container} >/dev/null 2>&1
     docker rm -fv ${container} >/dev/null 2>&1
-  else
+  elif [ "$OS" == "OSX" ]; then
     docker-machine rm -f ${project} >/dev/null 2>&1
```

Comment:
yeah that is intentional.

that said, when docker-native becomes the only supported impl this second conditional goes away all together ... thereby addressing the concern, at least as i read it.